src/AstClause.h

```diff
@@ -201,97 +191,40 @@ class AstClause : public AstNode {
         return head.get();
     }
 
-    /** Return the number of elements in the body of the Clause */
-    // TODO (b-scholz): remove this method
-    size_t getBodySize() const {
-        return atoms.size() + negations.size() + provNegations.size() + constraints.size();
-    }
-
-    /** Return the i-th Literal in body of the clause */
-    // TODO (b-scholz): remove this method
-    AstLiteral* getBodyLiteral(size_t idx) const {
-        if (idx < atoms.size()) {
-            return atoms[idx].get();
-        }
-        idx -= atoms.size();
-        if (idx < negations.size()) {
-            return negations[idx].get();
-        }
-        idx -= negations.size();
-        if (idx < provNegations.size()) {
-            return provNegations[idx].get();
-        }
-        idx -= provNegations.size();
-        return constraints[idx].get();
-    }
-
     /** Obtains a copy of the internally maintained body literals */
     std::vector<AstLiteral*> getBodyLiterals() const {
```

Comment:
Would be nice if this just returned a const ref to `bodyLiterals` but that's a much bigger change elsewhere :(